modules/qr_code.py

```diff
@@ -0,0 +1,55 @@
+import qrcode
+import uuid
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+class QRCode:
+    def __init__(self, base_url, qr_cache, max_size) -> None:
+        self.mapping = {}
+        self.base_url = base_url
+        self.qr_cache = qr_cache
```

Comment:
i really dont like nitpicking but can we do this as the variable name? just so we know it means a filepath
```suggestion
        self.qr_cache_path = qr_cache_path
```